lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -245,26 +245,26 @@ def self.get_release(api_token, owner_name, app_name, release_id)
         end
       end
 
-      # get distribution group
-      def self.get_group(api_token, owner_name, app_name, group_name)
+      # get distribution group or store
+      def self.get_destination(api_token, owner_name, app_name, destination_type, destination_name)
         connection = self.connection
 
         response = connection.get do |req|
-          req.url("/v0.1/apps/#{owner_name}/#{app_name}/distribution_groups/#{ERB::Util.url_encode(group_name)}")
+          req.url("/v0.1/apps/#{owner_name}/#{app_name}/distribution_#{destination_type}s/#{ERB::Util.url_encode(destination_name)}")
           req.headers['X-API-Token'] = api_token
           req.headers['internal-request-source'] = "fastlane"
         end
 
         case response.status
         when 200...300
-          group = response.body
-          UI.message("DEBUG: received group #{JSON.pretty_generate(group)}") if ENV['DEBUG']
-          group
+          destination = response.body
+          UI.message("DEBUG: received #{destination_type} #{JSON.pretty_generate(group)}") if ENV['DEBUG']
```

Comment:
Debug message should be fixed from `#{JSON.pretty_generate(group)}"` to `#{JSON.pretty_generate(destination)}"`

Reply:
That will fix tests

Reply:
Nice catch!